src/lib/components/event/event-summary-table.svelte

```diff
@@ -54,7 +62,11 @@
       </div>
     </div>
   </div>
-  <div class="table-header-row-responsive rounded-t-md">
+  <div
+    class="table-header-row-responsive rounded-t-md {typedError
+      ? 'header-hidden-responsive'
+      : ''}"
```

Comment:
```suggestion
    class="table-header-row-responsive rounded-t-md"
    class:header-hidden-responsive={typedError}
```

Reply:
Actually, I think you could just do `class:hidden={typedError}` for this one.

Reply:
`class:hidden={typedError}` didn't work 😞 But updated to your first suggestion, thank you!